docs/contributing/index.md

```diff
@@ -1,69 +1,94 @@
 
-It is recommended to read the [Contributing pages in ROS 2 Docs](https://docs.ros.org/en/rolling/Contributing.html) as well because Autoware's guidelines generally depend on them.
+## Code of Conduct
 
-For general information about OSS contributions, [Open Source Guides](https://opensource.guide/) by GitHub is a good resource.
+To ensure our community stays open and inclusive, our [code of conduct](https://github.com/autowarefoundation/autoware/blob/main/CODE_OF_CONDUCT.md) adheres to the [Contributor Covenant](https://www.contributor-covenant.org/) that has been adopted by [many other open source communities](https://www.contributor-covenant.org/adopters/). By participating in the Autoware community, you agree to follow this code.
 
-!!!info
+If you believe that someone in the community has violated the code of conduct, please make a report by emailing conduct@autoware.org.
 
-    If you want to make changes to the guidelines in this documentation site, you can open a new discussion thread and propose it.
+## What Should I Know Before I Get Started?
 
-## Code of conduct
+### Autoware concepts
 
-We welcome contributions from everyone.  
-To ensure our community stays open and healthy, we adhere to the [Contributor Covenant](https://www.contributor-covenant.org/), a widely used [code of conduct](https://github.com/autowarefoundation/autoware/blob/main/CODE_OF_CONDUCT.md) adopted by many other communities listed [here](https://www.contributor-covenant.org/adopters/).
+To gain a high-level understanding of Autoware's architecture and design, the following pages provide a brief overview:
 
-Everyone participating in the Autoware community is expected to follow the code of conduct.  
-If someone in the community happens to be violating these terms, let the maintainers know, and we will address it as soon as possible.
+- [Autoware architecture](https://autowarefoundation.github.io/autoware-documentation/main/design/)
+- [Autoware concepts](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-concepts/)
 
-!!!warning
+For experienced developers, the [Autoware interfaces](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/) and [individual component pages](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/components/) should also be reviewed to understand the inputs and outputs for each component or module at a more detailed level.
 
-    Note that contributions that ignore guidelines are not contributions but nuisances.
+### Contributing to open source projects
 
-## Contribution workflows
+If you are new to open source projects, we recommend reading GitHub's [How to Contribute to Open Source guide](https://opensource.guide/how-to-contribute/) for an overview of why people contribute to open source projects, what it means to contribute and much more besides.
```

Comment:
If possible, I'd like to add https://docs.ros.org/en/rolling/Contributing.html here.

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/91#discussion_r859272117

Reply:
This goes back to the reason why I removed the ROS 2 docs link. Ideally we should include the information that we think is relevant in our contributing guidelines, but at the very least we should indicate which specific information on that page should be reviewed by contributors? 

I'm guessing it's the Tenets section? 🤔

Reply:
Hmm, probably yes. I love the Tenets section!

Reply:
Okay. I think this might also have to be part of a follow-up PR. 

There are a lot of good points in the ROS 2 contribution tenets that can be included in our (😅 ) own contribution guidelines. However, I think the wording will need to be adjusted in a few places. 

For example, Open Robotics is described as being a gate-keeper. Although I understand the intent behind that description, the word `gate-keeper` has negative connotations (something that evokes a feeling or idea beyond the literal/primary meaning) so I would prefer to avoid using it.

Reply:
Thank you! I'll leave it to you. :pray: